packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -29,6 +30,8 @@ const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
 type Properties = ChatProperties & {
   title: string;
   showDayHeaders: boolean;
+  dayHeaderFormat: null | Format;
+  messageTimestampFormat: null | Format;
```

Comment:
https://github.com/Zedwag/DevExtreme/blob/9614145c4838d3944504d7fb2af44433f4d1d4e4/packages/devextreme/js/__internal/ui/date_box/m_date_box.base.ts#L80-L81

dateSerializationFormat option has `undefined` default value, why we should use null? If we will use `undefined` it should looks like
```
messageTimestampFormat?: Format;
```

Reply:
Here we should have the same types that are added in correspond d.ts file

Reply:
ed2574f9d5f3c5a301c284eb8acaba16c73321ad